docs/.vitepress/vitepress/components/common/vp-theme-toggler.vue

```diff
@@ -14,12 +14,12 @@ watch(
 )
 
 let resolveFn: (value: boolean | PromiseLike<boolean>) => void
-const switchTheme = (event: MouseEvent) => {
+const switchTheme = (event: MouseEvent | KeyboardEvent) => {
   const isAppearanceTransition =
     // @ts-expect-error
     document.startViewTransition &&
     !window.matchMedia('(prefers-reduced-motion: reduce)').matches
-  if (!isAppearanceTransition || !event) {
+  if (!isAppearanceTransition || !event || event instanceof KeyboardEvent) {
```

Comment:
The judgment here is not necessary. We should ensure that the transition effects are effective when switching themes using the keyboard and mouse.